### DISCUSSION TITLE
Doge qt not running :'(

### DISCUSSION THREAD

**Discussion by hashem8ali** [dogecoin/dogecoin#3699] [Q&A]
My doge QT was updating and syncing fine up until a few days ago. Got notification for a Java update, which required me to remove my current version of Java. I installed the newest version (32-bit version) and now my doge qt won't run. 
The message on my debug.txt is the following: 2024-11-15 07:29:36 LoadBlockIndexDB: transaction index disabled
I tried the following fix: System properties --> Environment Variables. Create a new variable under "system variables" called JAVA_HOME linked to the bin folder of Java. (C:\Program Files (x86)\Java\jre1.8.0_431\bin) This did not work. 
Appreciate your help or suggestions
![doge qt run error message](https://github.com/user-attachments/assets/b6709d4c-dfe6-45a4-9974-a3c74d2abc7e)
![doge qt run error fix attempt](https://github.com/user-attachments/assets/123f780d-6729-41bb-a3fa-192d74c1aa08)

**Comment by patricklodder** [comment]
Dogecoin Core has nothing to do with Java. Where did you get this notice? Is it possible it's unrelated?

The error given means something went wrong with pruning. I'm creating an issue from this discussion to track it down and fix it.

**Reply by patricklodder** [reply]
@hashem8ali I'm thinking right now that perhaps you needed the 64-bit version? Do you still have the old Qt binary you used? Are you sure it's 32-bit?

Reading a 64-bit data dir with 32-bit software may actually cause this (I'll try to reproduce this). I've seen some similar issues in the past

**Reply by hashem8ali** [reply]
@patricklodder thanks for the response. You definitely need the 32-bit version of Java and it's essential for running the original dogecoin core. Appreciate any other insights you could offer.

**Reply by patricklodder** [reply]
> You definitely need the 32-bit version of Java and it's essential for running the original dogecoin core.

Says who? This is an interesting claim.
